tools/local-sensor/scripts/README.md

```diff
@@ -0,0 +1,70 @@
+# Local-sensor
+
+## How to reproduce the performance tests
+
+### Using the `local-sensor.sh` script
+
+You can reproduce the test used to capture sensor's metrics easily by using the `local-sensor.sh` script.
```

Comment:
We will have a spreadsheet with the metrics once #3409, #3351, #3082, and this one are merged; but now that I think about it since it will be an internal document maybe it would make sense to rephrase this 🤔 Maybe something like `You can run reproducible tests to capture sensor's metrics easily by using [...]` WDYT?

Reply:
I rephrased the line and I added a TODO to add the link in a following PR once we have the spreadsheet as we discussed 🙂